src/nix.rs

```diff
@@ -194,22 +198,27 @@ impl CallOpts {
     /// # env::set_var("NIX_PATH", "nixpkgs=./nix/bogus-nixpkgs/");
     ///
     /// let tempdir = tempfile::tempdir().unwrap();
```

Comment:
Nice! Let's clean up the `tempdir` / `tempfile` references throughout this file.